src/ctia/stores/es/crud.clj

```diff
@@ -213,7 +218,10 @@ It returns the documents with full hits meta data including the real index in wh
                   (get-docs-with-indices conn-state [id] {})]
          (if (allow-write? current-doc ident)
            (let [update-doc (assoc realized
-                                   :id (ensure-document-id id))]
+                                   :id (ensure-document-id id))
+                 stored->es-stored (build-stored-transformer stored->es-stored stored-schema es-stored-schema
+                                                             {:op :update-record
+                                                              :prev (es-coerce! current-doc)})]
```

Comment:
@ereteog Here's where I'm grabbing the previous "raw" document so it can be compared with the patch. It's used in the `:stored->es-stored` coercion in `ctia.entity.incident/store-opts`.

I haven't fully checked if this is a good idea. But it seems promising.

Reply:
that seems nice!
`op` might not be necessary if we have `prev` (see my previous suggestion).